Sources/WeatherSensorService/Internal/MockWeatherSensorReader.swift

```diff
@@ -4,7 +4,6 @@
 //
 // Copyright © 2022 SpinDance. All rights reserved.
 //
-
```

Comment:
Typically you have a blank line here between file header and first import, please put undo this change.